client/webserver/site/src/js/state.ts

```diff
@@ -26,55 +37,68 @@ export default class State {
     return null
   }
 
-  /* dark sets the dark-mode cookie. */
-  static dark (dark: boolean) {
-    this.setCookie(darkModeCK, dark ? '1' : '0')
-    if (dark) {
-      document.body.classList.add('dark')
-    } else {
-      document.body.classList.remove('dark')
-    }
+  /*
+   * removeCookie tells the browser to stop using cookie. It's not enough to simply
+   * erase cookie value because browser will still send it to the server (with empty
+   * value), and that's not what server expects.
+   */
+  static removeCookie (cKey: string) {
+    document.cookie = `${cKey}=;expires=Thu, 01 Jan 1970 00:00:01 GMT;`
   }
 
   /*
    * isDark returns true if the dark-mode cookie is currently set to '1' = true.
    */
   static isDark () {
-    return document.cookie.split(';').filter((item) => item.includes(`${darkModeCK}=1`)).length
+    return document.cookie.split(';').filter((item) => item.includes(`${State.DarkModeCK}=1`)).length
   }
 
   /* passwordIsCached returns whether or not there is a cached password in the cookies. */
   static passwordIsCached () {
-    return !!this.getCookie(pwKeyCK)
+    return !!this.getCookie(State.PwKeyCK)
   }
 
-  /* store puts the key-value pair into Window.localStorage. */
-  static store (k: string, v: any) {
-    window.localStorage.setItem(k, JSON.stringify(v))
+  /* showLeftMarketDock returns whether or not user wants left market doc shown. */
+  static showLeftMarketDock () {
+    return this.fetchLocal(State.LeftMarketDockLK) === '1'
   }
```

Comment:
Only used one place and is specific to **/markets**. I'd say just do this in **markets.ts** and forego the method.

Reply:
yeah, makes sense, applied